code/obj/item/gun/ammo.dm

```diff
@@ -475,13 +492,15 @@
 
 	smg
 		name = "9mm SMG magazine"
+		desc = "An extended 9mm magazine."
 		amount_left = 30.0
 		max_amount = 30.0
 		ammo_type = new/datum/projectile/bullet/bullet_9mm/smg
 
 /obj/item/ammo/bullets/nine_mm_NATO
-	sname = "9mm NATO"
+	sname = "9mm nonlethal" // the naming on this is so fucking stupid
```

Comment:
Considering these still do some bleed and damage, 'nonlethal' isn't a good term for them- after talking with some Gundevs, "9mm Rubber" or "9mm Polymer" would both be good.
also, for the description- maybe "Despite being designed to incapacitate,"